debezium-core/src/main/java/io/debezium/connector/common/BaseSourceTask.java

```diff
@@ -195,6 +195,7 @@ private void stop(boolean restart) {
             catch (InterruptedException e) {
                 Thread.interrupted();
                 LOGGER.error("Interrupted while stopping coordinator", e);
+                doStop();
```

Comment:
I'm not quite convinced of this. Do you have insight into *why* we see an interruption here to begin with?